x-pack/plugins/case/server/client/cases/create.test.ts

```diff
@@ -38,75 +39,100 @@ describe('create', () => {
         caseSavedObject: mockCases,
         caseConfigureSavedObject: mockCaseConfigure,
       });
       const caseClient = await createCaseClientWithMockSavedObjectsClient({ savedObjectsClient });
-      const res = await caseClient.client.create({ theCase: postCase });
+      const res = await caseClient.client.create(postCase);
 
-      expect(res).toEqual({
-        id: 'mock-it',
-        comments: [],
-        totalComment: 0,
-        closed_at: null,
-        closed_by: null,
-        connector: {
-          id: '123',
-          name: 'Jira',
-          type: ConnectorTypes.jira,
-          fields: { issueType: 'Task', priority: 'High', parent: null },
-        },
-        created_at: '2019-11-25T21:54:48.952Z',
-        created_by: { full_name: 'Awesome D00d', email: 'd00d@awesome.com', username: 'awesome' },
-        description: 'This is a brand new case of a bad meanie defacing data',
-        external_service: null,
-        title: 'Super Bad Security Issue',
-        status: CaseStatuses.open,
-        tags: ['defacement'],
-        updated_at: null,
-        updated_by: null,
-        version: 'WzksMV0=',
-        settings: {
-          syncAlerts: true,
-        },
-      });
+      expect(res).toMatchInlineSnapshot(`
+        Object {
+          "closed_at": null,
+          "closed_by": null,
+          "comments": Array [],
+          "connector": Object {
+            "fields": Object {
+              "issueType": "Task",
+              "parent": null,
+              "priority": "High",
+            },
+            "id": "123",
+            "name": "Jira",
+            "type": ".jira",
+          },
+          "created_at": "2019-11-25T21:54:48.952Z",
+          "created_by": Object {
+            "email": "d00d@awesome.com",
+            "full_name": "Awesome D00d",
+            "username": "awesome",
+          },
+          "description": "This is a brand new case of a bad meanie defacing data",
+          "external_service": null,
+          "id": "mock-it",
+          "settings": Object {
+            "syncAlerts": true,
+          },
+          "status": "open",
+          "subCases": undefined,
+          "tags": Array [
+            "defacement",
+          ],
+          "title": "Super Bad Security Issue",
+          "totalAlerts": 0,
+          "totalComment": 0,
+          "type": "individual",
+          "updated_at": null,
+          "updated_by": null,
+          "version": "WzksMV0=",
+        }
+      `);
 
       expect(
         caseClient.services.userActionService.postUserActions.mock.calls[0][0].actions
-      ).toEqual([
-        {
-          attributes: {
-            action: 'create',
-            action_at: '2019-11-25T21:54:48.952Z',
-            action_by: {
-              email: 'd00d@awesome.com',
-              full_name: 'Awesome D00d',
-              username: 'awesome',
+        // using a snapshot here so we don't have to update the text field manually each time it changes
```

Comment:
I like the idea. I am a bit sceptical with it because sometimes we just press `Update snapshots` to make jest happy without investigating why the snapshot changed.

Reply:
We chatted about this and I think we're going to stick with inline snapshots for now so we can see the changes.